src/Utils/HtmlTable.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+namespace SMW\Utils;
+
+use Html;
+
+/**
+ * @license GNU GPL v2+
+ * @since 3.0
+ *
+ * @author mwjames
+ */
+class HtmlTable {
+
+	/**
+	 * @since 3.0
+	 *
+	 * @param string $html
+	 * @param array $attributes
+	 *
+	 * @return string
+	 */
+	public static function table( $html = '', array $attributes = array() ) {
```

Comment:
Well, its better than having `span` and `div`'s being sprinkled all over the code. Nothing wrong on using statics where an instance isn't required.